env0/resource_variable_set_assignment.go

```diff
@@ -0,0 +1,204 @@
+package env0
+
+import (
+	"context"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type variableSetAssignmentSchema struct {
+	Scope   string
+	ScopeId string
+	SetIds  []string
+}
+
+func resourceVariableSetAssignment() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceVariableSetAssignmentCreate,
+		UpdateContext: resourceVariableSetAssignmentUpdate,
+		ReadContext:   resourceVariableSetAssignmentRead,
+		DeleteContext: resourceVariableSetAssignmentDelete,
+
+		Schema: map[string]*schema.Schema{
+			"scope": {
+				Type:             schema.TypeString,
+				Description:      "the resource(scope) type to assign to. Valid values: 'template', 'environment', 'workflow', 'organization', 'project'",
```

Comment:
the allowed values are: organization, project, template, module, environment, deployment
https://docs.env0.com/reference/configuration-find-set-assignments-by-scope see what the assignmentScope is allowed to be.

Reply:
thank you. will fix.